Phonebook.Frontend/index_rehash.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
What do this file?

Reply:
It recalculates the hash of the index.html after updating the environment variables in the file.